packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -204,13 +251,15 @@ export class HttpPlugin extends classes.BasePlugin {
    * @param original The original patched function.
    * @param args The arguments to the original function.
    */
-  // tslint:disable:no-any
-  makeRequestTrace(request: any, args: any, plugin: HttpPlugin): any {
-    return (span: types.Span) => {
+  // disable-next-line to disable no-any check for args parameter
+  // tslint:disable-next-line
```

Comment:
`tslint:disable-next-line:no-any`

Reply:
Though, it looks like you never need to use anything other than `args[0]`. Why not just pass the first argument in?

Reply:
Agree.

Reply:
Nit: The type of `options` should now be known (isn't it `RequestOptions | string | URL`)

Reply:
Some compiler and/or tests issues. A TODO was added.